build.gradle

```diff
@@ -12,3 +12,9 @@ allprojects {
         jcenter()
     }
 }
+
+apply plugin: 'checkstyle'
+
+checkstyle {
+    configFile = new File("${project.projectDir}", "checkstyle.xml")
+}
```

Comment:
also you should probably put this block into the `allprojects` block above to apply the plugin to all subprojects